go/apps/simpleexample/SimpleClientCpp/helpers.cc

```diff
@@ -26,7 +26,7 @@
 #include <string>
 #include <thread>
 
-#include <taosupport.pb.h>
```

Comment:
Yea that's a good question and I'm not sure.. We could reference them to the directory or perhaps have symlinks?